source/includes/code-examples/DeleteOne.cs

```diff
@@ -0,0 +1,77 @@
+using MongoDB.Bson;
+using MongoDB.Bson.Serialization.Attributes;
+using MongoDB.Bson.Serialization.Conventions;
+using MongoDB.Driver;
+using static System.Console;
+
+namespace CSharpExamples.UsageExamples;
+
+public class DeleteOne
+{
+    private static IMongoCollection<Restaurant> _restaurantsCollection;
+    private static string _mongoConnectionString = "<Your MongoDB URI>";
+
+    public static void Main(string[] args)
+    {
+        Setup();
+
+        var doc = _restaurantsCollection.Find(Builders<Restaurant>.Filter
+            .Eq(r => r.Name, "Ready Penny Inn")).First();
+
+        // Delete a document using builders
+        WriteLine("Deleting a document with builders...");
+        var result = DeleteARestaurantBuilder();
+
+        WriteLine($"Deleted documents: {result.DeletedCount}");
+
+        Restore(doc);
+    }
+
+    private static DeleteResult DeleteARestaurantBuilder()
+    {
+        // start-delete-one-builders
+        var filter = Builders<Restaurant>.Filter
+            .Eq(r => r.Name, "Ready Penny Inn");
+
+        var result = _restaurantsCollection.DeleteOne(filter);
```

Comment:
better just to use `return _restaurantsCollection.DeleteOne(filter);` Same in other places